### ISSUE TITLE
Voie présente dans BAL mais manquante dans pifomètre

### ISSUE THREAD

**Issue by LySioS** [osm-fr/bano#306]
commune : La Cabanasse (n° INSEE 66027)
voie concernée : Rue Moffre
adresses associées à la rue : oui

**Comment by Olyon** [comment]
https://adresse.data.gouv.fr/base-adresse-nationale/66027_5500_00006
https://bano.openstreetmap.fr/pifometre/index.html#insee=66027&tab=2
La clé d'interopérabilité correspond a un code Fantoir 660275500W lié au nom "rue Joffre" dans le Fantoir.
Le problème vient peut être de là.

**Comment by LySioS** [comment]
Autre commune mais problème similaire, je ne sais pas s'il faut ouvrir un nouveau ticket (?)

commune: La Llagonne (66098)
BAL 100% : https://adresse.data.gouv.fr/commune/66098
endroits concernés : (renseignés comme des voies dans la BAL mais plutôt des lieux-dits/hameaux en réalité)
* Lo Moli https://www.openstreetmap.org/search?query=llagonne%20moli#map=19/42.51447/2.11201
* Los Colls https://www.openstreetmap.org/node/915282469#map=17/42.53042/2.12802
* Caugne de l'Ours https://www.openstreetmap.org/#map=19/42.52743/2.10930

**Comment by vdct** [comment]
> https://adresse.data.gouv.fr/base-adresse-nationale/66027_5500_00006 https://bano.openstreetmap.fr/pifometre/index.html#insee=66027&tab=2 La clé d'interopérabilité correspond a un code Fantoir 660275500W lié au nom "rue Joffre" dans le Fantoir. Le problème vient peut être de là.
Oui. Pour qu'une voie apparaisse dans Pifometre, il lui faut une des 2 conditions :
- un lien entre BAN et FANTOIR
- une absence de code FANTOIR dans la BAN
Ici la rue Moffre n'a aucun des 2 : 
- aucune rue Moffre dans FANTOIR (mais peut-être une faute côté FANTOIR avec 2 rues JOFFRE)
- mais un code Fantoir dans la BAN pour la rue Moffre (celui d'une des 2 rues Joffre en effet)
donc pas de rue Moffre dans Pifometre, sauf dans l'onglet des rues OSM non trouvées.
Soit ça reste en l'état, soit on décrète que le code 660275500W est en fait celui de la rue Moffre, on le renseigne dans le tag ref:FR:FANTOIR, et ça la fera apparaître dans l'onglet des rues rapprochées avec adresses.

**Comment by vdct** [comment]
> Autre commune mais problème similaire, je ne sais pas s'il faut ouvrir un nouveau ticket (?)
> 
> commune: La Llagonne (66098) BAL 100% : https://adresse.data.gouv.fr/commune/66098 endroits concernés : (renseignés comme des voies dans la BAL mais plutôt des lieux-dits/hameaux en réalité)
> 
>     * Lo Moli https://www.openstreetmap.org/search?query=llagonne%20moli#map=19/42.51447/2.11201
> 
>     * Los Colls https://www.openstreetmap.org/node/915282469#map=17/42.53042/2.12802
> 
>     * Caugne de l'Ours https://www.openstreetmap.org/#map=19/42.52743/2.10930

Depuis ton message tu les as intégrés en lieux-dits, et ils sont tous là : https://bano.openstreetmap.fr/pifometre/index.html#insee=66098&tab=5
